README.md

```diff
@@ -15,35 +16,63 @@ that you can use to tag your questions.
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1320/badge)](https://bestpractices.coreinfrastructure.org/projects/1320)
 ![Code style](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)
 [![Follow on Twitter](https://img.shields.io/twitter/follow/MaterialUI.svg?label=follow+Material-UI)](https://twitter.com/MaterialUI)
-
 [![PeerDependencies](https://img.shields.io/david/peer/mui-org/material-ui.svg)](https://david-dm.org/mui-org/material-ui#info=peerDependencies&view=list)
 [![Dependencies](https://img.shields.io/david/mui-org/material-ui.svg)](https://david-dm.org/mui-org/material-ui)
 [![DevDependencies](https://img.shields.io/david/dev/mui-org/material-ui.svg)](https://david-dm.org/mui-org/material-ui#info=devDependencies&view=list)
 
-> Material-UI is a set of [React](http://facebook.github.io/react/) components that implement
-[Google's Material Design](https://www.google.com/design/spec/material-design/introduction.html) specification.
+</div>
+
+<h2 align="center">Supporting Material-UI</h2>
+
+Material-UI is an MIT-licensed open source project. It's an independent project with ongoing development made possible thanks to the support of these awesome [backers](/BACKERS.md). If you'd like to join them, please consider:
+- [Become a backer or sponsor on Patreon](https://www.patreon.com/oliviertassinari).
+- [Become a backer or sponsor on OpenCollective](https://opencollective.com/material-ui).
+
+Through contributions, donations, and sponsorship, you allow Material-UI to thrive.
+Your donations directly support office hours, continued enhancements, great documentation and learning materials!
+
+### What's the difference between Patreon and OpenCollective?
 
-## We've moved!
+Funds donated via Patreon directly support [Olivier Tassinari](https://github.com/oliviertassinari)'s work on Material-UI.
+Funds donated via OpenCollective are managed with transparent expenses and will be used to compensate work and expenses by core team members.
```

Comment:
I would say, "are managed transparently and will be used to compensate work and expenses by core team members."